lib/util/git/index.spec.ts

```diff
@@ -208,6 +211,10 @@ describe('util/git/index', { timeout: 10000 }, () => {
       beforeEach(async () => {
         const repo = Git(base.path);
 
+        modifiedAuth.getGitEnvironmentVariables.mockReturnValue({
+          GIT_ALLOW_PROTOCOL: 'file',
```

Comment:
This was the minimal git config required for the tests to pass. Otherwise, this error is thrown 
> Error: fatal: not a git repository: ../.git/modules/submodule
fatal: could not reset submodule index

without this `getGitEnvironmentVariables` returns {}, which messes up the tests. This only seems to be required when running the unit tests, I guess, when running the code for real git inherits the required environment, or there are always HOST rules added, which might be sufficient to inform git that this is actually a git repository. I'm not sure on the underlying requirements of git